apps/admin-panel/app/ledger-account/ledger-account-page.stories.tsx

```diff
@@ -1,15 +1,23 @@
 import type { Meta, StoryObj } from "@storybook/react"
+import { MockedProvider } from "@apollo/client/testing"
 
 import LedgerAccountIndexPage from "./page"
 
 const LedgerAccountIndexPageStory = () => {
-  return <LedgerAccountIndexPage />
+  return (
+    <MockedProvider>
```

Comment:
I played with actually mocking a response but the mock provider was not behaving consistently. I can show what I had and would love to take a look at that with someone.